### PR TITLE
Re-adding docs for Fast.com

### PR DESCRIPTION
**Pull request by erwindouna**
## Proposed change

Update the ping integration documentation, added the config flow and removed the YAML configuration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/98686
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fastdotcom.markdown [hunk 1]
```diff
@@ -25,62 +26,12 @@ If you want to measure bandwidth metrics other then download such as ping and up
 
 Enabling this integration will automatically create the Fast.com Sensor.
 
-By default, a speed test will be run every hour. The user can change the update frequency in the configuration by defining the `scan_interval` for a speed test to run.
+By default, a speed test will be run every hour. The user can manually run a speed test via the `homeassistant.update_entity` service.
```
Comment (gjohansson-ST) [inline source/_integrations/fastdotcom.markdown:30]: I don't think that's the case as it's not using the `DataUpdateCoordinator`. I think they need to use the manual `update` service which is created by the integration

File: source/_integrations/fastdotcom.markdown [hunk 2]
```diff
@@ -8,6 +8,7 @@ ha_release: 0.88
 ha_iot_class: Cloud Polling
 ha_codeowners:
   - '@rohankapoorcom'
+  - '@erwindouna'
```
Comment (gjohansson-ST) [inline source/_integrations/fastdotcom.markdown:12]: Should add config_entry here in the config part

**Review by gjohansson-ST — CHANGES_REQUESTED** [review]

File: source/_integrations/fastdotcom.markdown (hunk 1 above)
Comment (erwindouna) [reply source/_integrations/fastdotcom.markdown:30]: It uses the `async_track_time_interval(hass, data.update, timedelta(hours=DEFAULT_INTERVAL))` to retry every hour.

File: source/_integrations/fastdotcom.markdown (hunk 2 above)
Comment (erwindouna) [reply source/_integrations/fastdotcom.markdown:12]: Care to elaborate on what is needed? This part is new for me.

File: source/_integrations/fastdotcom.markdown [hunk 3]
```diff
@@ -8,6 +8,7 @@ ha_release: 0.88
 ha_iot_class: Cloud Polling
```
Comment (gjohansson-ST) [inline source/_integrations/fastdotcom.markdown:8]:
```suggestion
ha_iot_class: Cloud Polling
ha_config_flow: true
```

File: source/_integrations/fastdotcom.markdown [hunk 4]
```diff
@@ -25,62 +27,12 @@ If you want to measure bandwidth metrics other then download such as ping and up
...
-  service: fastdotcom.speedtest
-```
+{% include integrations/config_flow.md %}
 
 ## Notes
 
 - When running on Raspberry Pi 3 or older, the maximum speed is limited by its 100 Mbit/s LAN adapter.
 - The sensor will return the maximum measured speed during a 15-second test.
-- Speed tests consume data depending on your internet speed, make sure to consider this if your internet connection has limited bandwidth.
+- Speed tests consume data depending on your internet speed, make sure to consider this if your internet connection has limited bandwidth.
```
Comment (c0ffeeca7) [inline source/_integrations/fastdotcom.markdown]:
```suggestion
- Speed tests consume data depending on your internet speed. Make sure to consider this if your internet connection has limited bandwidth.
```

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @erwindouna :+1:

**Review by gjohansson-ST — APPROVED** [review]

**Pull request merged**